src/utils.ts

```diff
@@ -62,16 +62,17 @@ export const getTemplateConfigByType = (
   content: string,
   darkMode: TailwindOptions['darkMode'],
   tailwindContent: TailwindOptions['content'],
+  extend: SdTailwindConfigType['extend'],
   plugins: string[]
 ) => {
+  const extendTheme = extend ? `theme: { extend: ${unquoteFromKeys(content, type)}, },` : `theme: ${unquoteFromKeys(content, type)},`
+
   const getTemplateConfig = () => {
     let config = `{
   mode: "jit",
   content: [${tailwindContent}],
   darkMode: "${darkMode}",
-  theme: {
-    extend: ${unquoteFromKeys(content, type)},
-  },`
+  ${extendTheme}`
```

Comment:
Looks good👍
However, when I run it in this state, the format of the generated `tailwind.config.js` is a bit broken.
I would appreciate the following change.
Please use the `sd.config.js` in the [example](https://github.com/nado1001/style-dictionary-tailwindcss-transformer/blob/main/example/with-css-variables/sd.config.js) and actually run it to check.

```
  const theme = extend
    ? `extend: ${unquoteFromKeys(content, type)},`
    : unquoteFromKeys(content, type)

  const getTemplateConfig = () => {
    let config = `{
  mode: "jit",
  content: [${tailwindContent}],
  darkMode: "${darkMode}",
  theme: {
    ${theme}
  },`
```